### ISSUE TITLE
ThrowIn direction is not handled correctly

### ISSUE THREAD

**Issue by mrbermell** [njustesen/botbowl#193]
The below test should pass but doesn't . Consequence is that throw in are never perpendicular to the line (i.e. D3 roll of 2). Something is wrong when ThrowIn procedure calculates the `x, y` direction from the D3 roll.  

This test needs to be run in PR #191 branch, where the very convenient `get_custom_game_turn()` function is implemented :smiley: 

P.S. how in the name of the Nuffle did I come across this because surely this bug never caused an error? Fun story, buy me a beer sometime! 

```python 
def test_crowd_surf_ball_carrier():
    game, blocker, ball_carrier = get_custom_game_turn(player_positions=[(5, 2)],
                                                       opp_player_positions=[(5, 1)],
                                                       ball_position=(5, 1))

    BBDie.fix(BBDieResult.PUSH)

    game.step(Action(ActionType.START_BLOCK, player=blocker))
    game.step(Action(ActionType.BLOCK, player=ball_carrier))
    game.step(Action(ActionType.SELECT_PUSH, position=Square(5, 0)))
    game.step(Action(ActionType.PUSH, position=Square(5, 0)))

    D6.fix(2)  # Crowd surf injury roll
    D6.fix(2)  # Crowd surf injury roll
    D3.fix(2)  # Throw in direction, should have vector (x,y) = (0,1) 
    D6.fix(2)  # Throw in distance
    D6.fix(2)  # Throw in distance
    D8.fix(7)  # Bounce direction with vector (x,y) = (0,1)

    # throw in is made from Square(5,1) and 
    # should go in y-direction for 2+2+1=5 step. Ending up at Square(5,6)
    final_square = Square(5,6) 

    game.step(Action(ActionType.FOLLOW_UP, position=blocker.position))  # Don't follow up. 

    assert game.has_report_of_type(OutcomeType.THROW_IN)
    assert game.get_ball().position == final_square
```

**Comment by njustesen** [comment]
I believe this is fixed in #191 . Can you confirm @mrbermell ?

**Comment by mrbermell** [comment]
Yes, this was fixed.
